packages/elpi-option-legacy-parser/elpi-option-legacy-parser.1.15.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" ]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/LPCIC/elpi"
+doc: "https://LPCIC.github.io/elpi/"
+dev-repo: "git+https://github.com/LPCIC/elpi.git"
+bug-reports: "https://github.com/LPCIC/elpi/issues"
+
+depends: [
+  "camlp5" {> "7.99"}
+  "ocaml" {< "4.14.0" }
+]
+synopsis: "ELPI - option for legacy parser"
+description: """
+Enables the -legacy-parser flag, implemented on top of CamlP5"""
+url {
+  src:
+    "https://github.com/LPCIC/elpi/releases/download/v1.15.1/elpi-1.15.1.tbz"
+  checksum: [
+    "sha256=eaea1f24532d9cb6bb7209725afca12955b27c4ff66ac945ca244ce792df427f"
+    "sha512=3a9faa5822593f672f7475ee8f2c0b03996031076ca79145b80e17b65f08ab0b68841daa2a36b7e046e9887683a01d47ac47c4a67820cdde2386d2c9ce81f204"
+  ]
+}
+x-commit-hash: "9e8c3abc8d102289ddc7fbbab6284319d952183b"
```

Comment:
If this package is purely virtual then the url is not needed
```suggestion
```

Reply:
OK, this this is what dune-release did. Is there a way to tell it to do so next time?

Reply:
I think you could at least move the virtual package's opam out of the project root? The copy in opam-repository will be enough. (If you ever bump it, it'll get trickier).